src/python/gudhi/point_cloud/timedelay.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
```

Comment:
Usually we start those source files with a notice that gives the license (MIT), the authors (you? plus someone else?) and the copyright holder (Fujitsu labs?). You can copy and adapt them from some other random file, say subsampling.pyx.

Reply:
Thank you for your comment.  
Please correct as follows.  
```
# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
# Author(s):       Martin Royer, Yuichi Ike, Masatoshi Takenouchi
#
# Copyright (C) 2019 Inria, Copyright (C) 2019 Fujitsu Laboratories Ltd.
# Modification(s):
#   - YYYY/MM Author: Description of the modification
```

Reply:
We could write 2020, although I don't mind 2019. You can continue editing your branch, commit a patch and push to takeshimeonerespect:master. It will automatically appear here if you do that.

Reply:
I fixed based on comment.